Tests/RoslynPluginGeneratorTests/ArchiveUpdaterTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ArchiveUpdaterTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarQube.Plugins.Test.Common;
+using System.IO;
+using System.IO.Compression;
+
+namespace SonarQube.Plugins.Roslyn.RoslynPluginGeneratorTests
+{
+    [TestClass]
+    public class ArchiveUpdaterTests
+    {
+        public TestContext TestContext { get; set; }
+
+        #region Tests
+
+        [TestMethod]
+        public void ArchiveUpdater_SimpleUpdateJar_Succeeds()
```

Comment:
Simple test for the zip updater: checks new files can be created in the root directory and sub-directories.